pwiz_tools/Skyline/Model/SmallMoleculeTransitionListReader.cs

```diff
@@ -1742,7 +1742,7 @@ private PeptideGroupDocNode GetMoleculePeptideGroup(SrmDocument document, Row ro
             if (String.IsNullOrEmpty(name))
                 name = document.GetSmallMoleculeGroupId();
             var metadata = new ProteinMetadata(name, String.Empty).SetWebSearchCompleted();  // FUTURE: some kind of lookup for small molecules
-            return new PeptideGroupDocNode(pepGroup, metadata, new[] { pep });
+            return new PeptideGroupDocNode(pepGroup, metadata, new[] { pep }, false);

```

Comment:
I wonder whether we should just leave this as true, since having automanage turned on for a Molecule List never affects anything.

Reply:
Not for the foreseeable future, no. But I feel like having it off kind of reinforces that expectation.